css/styles.css

```diff
@@ -226,10 +245,31 @@ header .link-set:last-child {
   display: flex;
   justify-content: space-between;
 }
-
 .job .title-big {
   margin-bottom: 50px;
 }
+.examples-thumb {
+  position: relative;
+}
+.examples-thumb > h3 {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  margin: 0;
+  width: 370px;
+  height: 70px;

```

Comment:
it's better not to use `height` in blocks with content. Use paddings (and left-right too, to prevent text from sticking to edge). Also you dont need `flex` here, just use `text-aligh.`